src/test/java/org/symphonyoss/symphony/messageml/elements/TextAreaTest.java

```diff
@@ -0,0 +1,179 @@
+package org.symphonyoss.symphony.messageml.elements;
+
+import static org.junit.Assert.assertEquals;
+import static org.symphonyoss.symphony.messageml.elements.TextArea.VALID_VALUES_FOR_REQUIRED_ATTR;
+
+import org.junit.Test;
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+
+public class TextAreaTest extends ElementTest {
+
+  private static final String NAME_ATTR = "name";
+  private static final String REQUIRED_ATTR = "required";
+  private static final String PLACEHOLDER_ATTR = "placeholder";
+
+  private static final String NAME_VALUE = "A name";
+  private static final String PLACEHOLDER_VALUE = "A placeholder";
+  private static final String INITIAL_VALUE = "An initial value";
+
+  @Test
+  public void testTextAreaWithRequiredAttributesOnly() throws Exception {
+    String input = String.format("<messageML><form><textarea name=\"%s\"></textarea></form></messageML>", NAME_VALUE);
+
+    context.parseMessageML(input, null, MessageML.MESSAGEML_VERSION);
+    Element messageML = context.getMessageML();
+    Element form = messageML.getChildren().get(0);
+    Element textArea = form.getChildren().get(0);
+
+    assertEquals(Form.class, form.getClass());
+    assertEquals(TextArea.class, textArea.getClass());
+
+    verifyTextAreaPresentationML((TextArea) textArea, false, false, false);
```

Comment:
As a pattern, these kind of _verify_ methods make it quite hard to understand what the expected output is. Consider refactoring the tests so that the expected PresentationML, Markdown and text are right there in the individual test methods.